# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/ntp/includes-fix.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---


Needed to compile ntp 4.2.0 ...

  - alan.rock@glaramara.freeserve.co.uk

diff -Naur ntp-4.2.0/include/ntp_md5.h ntp-4.2.0.new/include/ntp_md5.h
--- ntp-4.2.0/include/ntp_md5.h	2003-07-17 11:27:16.000000000 +0100
+++ ntp-4.2.0.new/include/ntp_md5.h	2003-11-11 13:31:52.000000000 +0000
@@ -3,6 +3,7 @@
  */
 
 #ifdef HAVE_MD5_H
+# include <sys/types.h>
 # include <md5.h>
 #else
 # include "rsa_md5.h"


